ds_autotuning_prototype/core_api/ffn_example_ds_launcher/zero_search_config.json

```diff
@@ -0,0 +1 @@
+{"1": {"reduce_bucket_size": [20000000.0, 2000000000.0, 20000000000.0], "allgather_bucket_size": [20000000.0, 2000000000.0, 20000000000.0]}, "2": {"allgather_bucket_size": [30000000.0, 3000000000.0, 30000000000.0]}}
```

Comment:
Is this necessary or could we remove this? To the best of my understanding it's just a duplicate of the yaml by the same name

Reply:
it's a test that you can provide the same config with either a `json` or `yaml` file. All of the examples are just scratchwork at this point, basically.